### PR TITLE
Align server_theme breakpoints configuration with tailwind breakpoints

### PR DESCRIPTION
**Pull request by bboro**
This aligns the breakpoints config with tailwind breakpoints.

Updated readme to highlight important point on setup.

### CODE REVIEW

**Comment by amitaibu** [comment]
To confirm, are we alreadu using the responsive image in the Article PEVB?

**Comment by amitaibu** [comment]
I mean here - https://github.com/Gizra/drupal-starter/blob/2d16d8a1d060c4ad21ba23510748a8405f96b722/web/modules/custom/server_general/src/EntityViewBuilder/NodeViewBuilderAbstract.php#L76-L84 I think it should change?

**Comment by bboro** [comment]
@amitaibu that looks like it passes the image's URL to the twig, so I highly doubt it's a responsive imageset with media queries etc. I don't think we're currently using any responsive image on drupal starter. This is just a bit of needed setup.

**Comment by amitaibu** [comment]
@bboro In that case could you please wire it, so it's used.

**Comment by bboro** [comment]
* Enabled Media and Responsive image
* Replaced Image reference `field_image` with Media reference `field_featured_image` in article node type
* Updated migration for articles, added media migration
* Updated hero frontend
* Added hero image styles and responsive image styles
* Wire up NodeArticle PEVB plugin with the new responsive image output
* Added `child-object-cover` and `child-object-contain` utility classes

| 640px (sm) | 768 (md) | 1024 (lg) | 1280 (xl) | 1920 |
| - | - | - | - | - |
| Screen Shot 2021-09-06 at 17 27 11 | Screen Shot 2021-09-06 at 17 27 17 | Screen Shot 2021-09-06 at 17 27 22 | Screen Shot 2021-09-06 at 17 27 29 | Screen Shot 2021-09-06 at 17 27 42 |

**Comment by bboro** [comment]
Markup output of responsive image:

image

File: README.md [hunk 1]
```diff
@@ -63,6 +63,21 @@ The directory structure:
 
 For theme development, it's advisable to entirely turn off caching: https://www.drupal.org/node/2598914
 
+### Breakpoints and Responsive Images
+
+It is advised to use Drupal's Responsive image module.
```
Comment (amitaibu) [inline README.md:68]: Can you add a short paragraph about how you decide on the measurements (set the browser width, measure the img, etc')

File: README.md [hunk 2]
```diff
@@ -63,6 +63,21 @@ The directory structure:
...
+
+If there are new breakpoints added, or existing breakpoints updated in
+`server_theme/tailwind.config.js`, you must ensure to also update the drupal
```
Comment (amitaibu) [inline README.md:71]:
```suggestion
`server_theme/tailwind.config.js`, you must ensure to also update the Drupal
```

**Review by amitaibu — APPROVED** [review]
Looks great, two small comments

File: README.md (hunk 1 above)
Comment (bboro) [reply README.md:68]: Added

**Pull request merged**